src/OutboundServer/api.yaml

```diff
@@ -1,940 +1,918 @@
-openapi: 3.0.1
-info:
-  title: Mojaloop SDK Outbound Scheme Adapter API
-  description: >
-    Specification for the Mojaloop SDK Scheme Adapter Outbound Transfers API
-
```

Comment:
These empty lines provide a way to split the description into multiple paragraphs. Do you want it to be in a single paragraph? By the way, you could try to render OpenAPI files using https://editor.swagger.io/

Reply:
I think I'm fine the way it is split now. Thanks.